src/routes/stacks/[stackId]/+page.server.ts

```
@@ -7,20 +9,56 @@
+			originalEnvContent: stack.envContent || ''
 		};
 
+		const settings = await getSettings();
+
+		const servicePorts: Record<string, string[]> = {};
+		if (stack.services) {
+			for (const service of stack.services) {
+				if (service.id) {
+					try {
+						const containerData = await getContainer(service.id);
+						if (containerData && containerData.networkSettings?.Ports) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)